types/jasmine/index.d.ts

```diff
@@ -980,6 +980,11 @@ declare namespace jasmine {
          */
         fullName: string;
 
+        /**
+         * The name of the file the suite was defined in.
+         */
+        filename: string;
```

Comment:
What version was this introduced in? Jasmine is at 5.4, but DT only says 5.1.

Reply:
This was included since 4.6.0 ([release notes](https://github.com/jasmine/jasmine/blob/main/release_notes/4.6.0.md)) and had been missed. This PR's focus is not to update all types to the latest version but add some missing types.